zetaclient/chains/base/observer.go

```diff
@@ -204,6 +211,21 @@ func (ob *Observer) WithLastBlockScanned(blockNumber uint64) *Observer {
 	return ob
 }
 
+// LastTxScanned get last transaction scanned.
+func (ob *Observer) LastTxScanned() string {
+	ob.mu.Lock()
```

Comment:
We don't need mutex here

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/2743f9dc1b608f81928dc99e067e9f65fcc6d716